examples/0_Simple/simpleStreams/host.gpr

```diff
@@ -1,4 +1,4 @@
-with "cuda_api_host.gpr";
+with "cuda_api_host";
```

Comment:
The GPR User's Guide recommend leaving the ".gpr" extensions. What do you think about adding the extensions where they are missing in other project files instead?

Reply:
We need to be consistent :-) All for going with the GPR UG recommendation.